modules/ROOT/pages/ai-azure.adoc

```diff
@@ -0,0 +1,215 @@
+= Azure OpenAI Service integration guide
+:pluginname: AI Assistant
+:plugincode: ai
+:plugincategory: premium
+:navtitle: Azure OpenAI Service integration guide
+:description_short: {pluginname} with Azure OpenAI Service
+:description: A guide for integrating {pluginname} plugin using Azure OpenAI Service.
+:keywords: example, demo, custom, plugin, ai, assistant, guide, azure, openai, service
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+== Introduction
+
+This guide provides instructions for integrating the {pluginname} plugin using the https://learn.microsoft.com/en-us/azure/ai-services/openai/[Azure OpenAI Service] in {productname}.
+
+Azure OpenAI Service provides REST API access to OpenAI's language models with Azure specific features such as private networking, regional availability, and responsible AI content filtering.
+
+Azure OpenAI co-develops the APIs with OpenAI. Therefore, the following examples should be compatible if using Azure OpenAI REST API endpoints or OpenAI API endpoints.
```

Comment:
The OpenAI integration is being separated out to its own page now, and credentials are non-transferrable, so we probably don't need to mention this. Future updates could also make them incompatible.